src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/subheadline/index.test.js

```diff
@@ -51,4 +51,18 @@ describe('convertToSubheadline', () => {
       expect(await convertToSubheadline(input)).toEqual(expected);
     });
   });
+  it(`should not convert a non CPS heading block to Optimo format`, async () => {
+    const input = {
+      model: {
+        copyrightHolder: 'Joe Maher',
+        height: 549,
+        locator: '729E/test/_63724392_gettyimages-1098075358.jpg',
+        originCode: 'cpsdevpb',
+        width: 976,
+      },
+      type: 'rawImage',
+    };
+
+    expect(await convertToSubheadline(input)).toBeNull();
```

Comment:
Just noticing that the `console.log` causes this to be output to the console when the tests are run:
```
  console.log src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/subheadline/index.js:12
    Incorrent block type rawImage
```
Wondering if there is a trivial way of suppressing this to make the test output cleaner?

Reply:
I'm not immediately sure how you would do that? Any ideas anyone?

Reply:
@andrewscfc if we add a mock like
```
global.console.log = jest.fn(); 
```
in the `it` block of the test, I don't think it will be logged.
We could then add another assertion that `global.console.log` is called

Reply:
Thanks @rhenshaw56 , I'll give that a go 😄

Reply:
Should be resolved by 8e32e28ff37380c247b6c5d3a9b4c54dcfd1a64a